_benchmark/user-guide/target-throughput.md

```diff
@@ -18,12 +18,12 @@ OpenSearch Benchmark has two testing modes, both of which are related to through
 
 When you do not specify a `target-throughput`, OpenSearch Benchmark latency tests are performed in *benchmarking mode*. In this mode, the OpenSearch client sends requests to the OpenSearch cluster as fast as possible. After the cluster receives a response from the previous request, OpenSearch Benchmark immediately sends the next request to the OpenSearch client. In this testing mode, latency is identical to service time.
 
+OpenSearch Benchmark issues one request at a time for a single-client thread, which is specified as `search-clients` in the workload parameters. If `target-throughput` is set to `0`, then OpenSearch Benchmark issues a request immediately after it receives the response from the previous request. If the `target-throughput` is not set to `0`, then OpenSearch Benchmark issues the next request in accordance with the `target-throughput`, assuming that responses are returned instantaneously.
+
 ## Throughput-throttled mode
 
 **Throughput** measures the rate at which OpenSearch Benchmark issues requests, assuming that responses will be returned instantaneously. However, users can set a `target-throughput`, which is a common workload parameter that can be set for each test and is measured in operations per second.
```

Comment:
Should we move "**Throughput** measures the rate at which OpenSearch Benchmark issues requests, assuming that responses will be returned instantaneously. " to be before line 10 and then remove "However, users can set a `target-throughput`, which is a common workload parameter that can be set for each test and is measured in operations per second." since it is already mentioned in the first paragraph?

Reply:
We can then just insert this paragraph:
When target-throughput is not set to `0`, OpenSearch Benchmark issues the next request in accordance with the `target-throughput`, assuming that responses are returned instantaneously.